vite.config.ts

```diff
@@ -10,36 +9,23 @@ import { VitePWA } from 'vite-plugin-pwa';
 import { createHtmlPlugin } from 'vite-plugin-html';
 import { Target, viteStaticCopy } from 'vite-plugin-static-copy';
 
+import { initSettings } from './scripts/build-tools/settings';
+
 export default ({ mode, command }: ConfigEnv): UserConfigExport => {
   // Shorten default mode names to dev / prod
   // Also differentiates from build type (production / development)
   mode = mode === 'development' ? 'dev' : mode;
   mode = mode === 'production' ? 'prod' : mode;
 
-  const localFile = `ini/.webapp.${mode}.ini`;
-  const templateFile = `ini/templates/.webapp.${mode}.ini`;
-
-  // The build ONLY uses .ini files in /ini to include in the build output.
-  // All .ini files in the directory are git ignored to customer specific values out of source control.
-  // However, this script will automatically create a .ini file for the current mode if it doesn't exist
-  // by copying the corresponding mode file from the ini/templates directory.
-  if (!fs.existsSync(localFile) && fs.existsSync(templateFile)) {
-    fs.copyFileSync(templateFile, localFile);
-  }
+  initSettings(mode);
 
   // Make sure to builds are always production type,
   // otherwise modes other than 'production' get built in dev
   if (command === 'build') {
     process.env.NODE_ENV = 'production';
   }
 
-  const fileCopyTargets: Target[] = [
-    {
-      src: localFile,
-      dest: '',
-      rename: '.webapp.ini',
-    },
-  ];
+  const fileCopyTargets: Target[] = [];
```

Comment:
I think we need to keep the localFile copy target. The ini file is meant to be a runtime modifyable place to edit settings.